third_party/tensorrt/local/BUILD

```diff
@@ -67,11 +67,6 @@ cc_import(
         ":windows": "lib/nvinfer.dll",
         "//conditions:default": "lib/x86_64-linux-gnu/libnvinfer.so",
     }),
-    static_library = select({
```

Comment:
Id still like to solve this correctly. Can you take a look at the places where linkstatic is used? I think thats the root cause.

Reply:
It throws undefined references to all the TensorRT symbols on Jetson AGX

Reply:
lets then break out the static library into a different target,

Reply:
separated the static library target